catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSource.java

```diff
@@ -316,6 +321,7 @@ public void refresh(Map<String, Object> configuration) throws SecurityServiceExc
       this.username = usernameValue;
       this.disableCnCheck = disableCnCheckProp;
       this.coordinateOrder = coordOrder;
+      this.allowRedirects = allowRedirects;
```

Comment:
If this changes... we need to recreate the clientFactory.  Same true if the username or password changes for that matter

Reply:
I think the factory is recreated on the following line https://github.com/codice/ddf/pull/2913/files#diff-914efc644076bea5db1b180837d0b485R325

Reply:
it is... but we don't even get here if the url and disableNcCheckProp didn't change.  We need to do this stuff if username, password, or allowRedirects changes... 

I am guessing username/password was added and this was missed.  But the if should be looking for a change in any of the 5 variables to force a recreate

Reply:
👍  agreed

Reply:
Good catch @millerw8, fixing now.